src/main/java/de/refactoringbot/rest/ConfigurationController.java

```diff
@@ -156,43 +117,39 @@
 	@ApiOperation(value = "Delete Git-Configuration with configuration id")
 	public ResponseEntity<?> deleteConfig(@PathVariable("configurationId") Long configurationId) {
 		// Check if configuration exists
-		Optional<GitConfiguration> existsConfig;
+		GitConfiguration config = null;
```

Comment:
the whole decission logic about how to handle the `GitConfiguration` could also be moved into the `configServie`, so the controller only delegates the call to te `configService.delteConfig(configId)` and handles the result.

Reply:
@Simulant87 : I think I addressed everything you requested in your review. Check my latest 2 commits.